examples/booksapp/booksapp_extension.yaml

```diff
@@ -1,6 +1,7 @@
 openapi: 3.0.1
 x-kusk:
   namespace: booksapp
+  host: "*"
```

Comment:
I think this should be an array, not the string. I.e. if there is the need to support multiple hostnames for the same service (example.org, example.com). Not required for Ambassador support specifically though.